diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -451,6 +451,11 @@ namespace diff_drive_controller{
         return;
       }
 
+      if(std::isnan(command.angular.z) || std::isnan(command.linear.x))
+      {
+        ROS_WARN("Received NaN in geometry_msgs::Twist. Ignoring command.");
```

Comment:
This should probably be throttled to avoid 200Hz prints if someone spams the controllers